project/AppDependencies.scala

```diff
@@ -0,0 +1,34 @@
+import sbt._
+
+object AppDependencies {
+  import play.core.PlayVersion
+  import play.sbt.PlayImport._
+
+
+  val bootstrapPlayVersion = "1.6.0"
+
+  val compile = Seq(
+    ws,
+    "uk.gov.hmrc"               %% "bootstrap-play-26" % bootstrapPlayVersion,
+    "org.apache.httpcomponents" %  "httpclient"        % "4.5.11"
+  )
+
+  def test(scope: String = "test,it") = Seq(
+    "uk.gov.hmrc"            %% "bootstrap-play-26"           % bootstrapPlayVersion % scope classifier "tests",
```

Comment:
bootstrap-play-26 is a compile time dependency, so shouldn't need to be repeated here

Reply:
good spot, will remove